databricks_cli/pipelines/cli.py

```diff
@@ -208,17 +213,45 @@ def reset_cli(api_client, pipeline_id):
 @provide_api_client
 def run_cli(api_client, pipeline_id):
     """
+    (Deprecated) "run" command is deprecated, please use "start" command instead.
+
     Starts the execution of a delta pipelines run by starting the cluster and processing data.
 
     Usage:
 
     databricks pipelines run --pipeline-id 1234
     """
+    click.echo("DeprecationWarning: \"run\" command is deprecated," +
+               " please use \"start\" command instead")
     _validate_pipeline_id(pipeline_id)
-    PipelinesApi(api_client).run(pipeline_id)
+    PipelinesApi(api_client).start(pipeline_id, full_refresh=False)
     click.echo("Run triggered for pipeline {}".format(pipeline_id))
 
 
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Starts the execution of a delta pipeline run')
+@click.option('--pipeline-id', default=None, type=PipelineIdClickType(),
+              help=PipelineIdClickType.help)
+@click.option('--full-refresh', default=False, type=bool,
+              help='If true, truncates tables and creates new checkpoint' +
+                   ' folders so data is reprocessed from scratch')
+@debug_option
+@profile_option
+@pipelines_exception_eater
+@provide_api_client
+def start_cli(api_client, pipeline_id, full_refresh):
+    """
+    Starts the execution of a delta pipelines run by starting the cluster and processing data.
+
+    Usage:
+
+    databricks pipelines start --pipeline-id 1234 --full-refresh=(false|true)
+    """
+    _validate_pipeline_id(pipeline_id)
+    PipelinesApi(api_client).start(pipeline_id, full_refresh=full_refresh)
+    click.echo("Update triggered for pipeline {}".format(pipeline_id))
```

Comment:
Maybe something like "Started an update for pipeline ..."?

Reply:
So we do use "update" here, which would seem to add to the argument for using consistent terminology.

Reply:
updated to "Started an update .."

Good point, switched the command visible to the user as "update" instead of "start"

Reply:
just fyi, reverted this from update to start, see my other comment.